### PR TITLE
ocp4 - CA-9 - update from partial to not applicable

### PR DESCRIPTION
**Pull request by itewk**
peer reviewed
* @kevindoingbizas

### CODE REVIEW

File: openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml [hunk 1]
```diff
@@ -302,18 +302,16 @@
...
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - key: a
       text: |
-        Engineering progress can be tracked at:
-
-        https://issues.redhat.com/browse/CMP-256
+        CA-9 is an organizational control outside the scope
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml]:
```suggestion
        Organizational processes to authorize internal connections of
        organization-defined information system components or classes of
        components to the information system is an organizational control outside the scope
```

Comment (shawndwells) [reply openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml]: To be consistent, and address user feedback, ``not applicable`` control responses should be more verbose and include text from the NIST control.

File: openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml [hunk 2]
```diff
@@ -302,18 +302,16 @@
...
+        of OpenShift configuration.
     - key: b
       text: |
-        Engineering progress can be tracked at:
-
-        https://issues.redhat.com/browse/CMP-256
+        CA-9 is an organizational control outside the scope
+        of OpenShift configuration.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml]: The language of CA-9(b) at https://nvd.nist.gov/800-53/Rev4/control/ca-9:

``b. Documents, for each internal connection, the interface characteristics, security requirements, and the nature of the information communicated.``

We'll need to document what ports and protocols OpenShift uses. Is this something you could help with? Eg how the cluster elements connect to each other, over what protocols, what kind of data is shared, etc.

**Comment by shawndwells** [comment]
Hi @kevindoingbizas and @itewk - Checking in on this PR. What's your take on the PR feedback? Are there existing resources that enumerate the ports and protocols that OCP uses? They'd be especially useful for CA-4(9)  (aside: @itewk this is part of the data needed to auto generate that gdoc you showed me)

**Comment by itewk** [comment]
Heyo team, can you take this over working with @shawndwells please.

@curt-matthews
@mradecker
@kevindoingbizas
@thamill
@mtclemente
@kpeeples
@dwinchell
@mammerman
@mlcassatt
@siliconjesus
@mertel-rh
@kevin-anderson-redhat

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**